src/Components/QuickTimesDropdown/QuickTimesDropdown.types.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    IDropdownOption,
+    IDropdownProps,
+    IRenderFunction,
+    IStyle,
+    IStyleFunctionOrObject,
+    ITheme
+} from '@fluentui/react';
+import { QuickTimeSpanKey } from '../../Models/Constants/Enums';
+
+export interface IQuickTimesDropdownProps {
+    defaultSelectedKey?: QuickTimeSpanKey;
+    hasLabel?: boolean;
+    label?: string;
```

Comment:
since there is a default label "Time span" within the component, this is additional check to see the intention if we want to show it or not:
![image](https://user-images.githubusercontent.com/45217314/197855974-ea4d5404-bec5-4991-b0d8-e9ae54235055.png)